app/Integrations/Packagist/Packages.php

```diff
@@ -171,7 +171,7 @@ public function download(array $package)
         $meta = stream_get_meta_data($temp);
         $path = $meta['uri'];
 
-        ($zip = new ZipArchive())->open($path);
+        with($zip = new ZipArchive())->open($path);
```

Comment:
It makes sense when you think about it tho. The lexer in <7.0 is shit (in comparison to 7.0, that is).